typescript-primitives-and-variables/main.ts

```diff
@@ -0,0 +1,19 @@
+const fullName: string = 'Brendan Kormos';
+const isCool: boolean = true;
+const totalPets: number = 2;
+console.log('fullname', fullName);
+console.log('fullname typeof', typeof fullName);
+console.log('iscool', isCool);
+console.log('iscool typeof', typeof isCool);
+console.log('totalpets ', totalPets);
+console.log('totalpets typeof', typeof totalPets);
+
+const unknown: null = null;
+
+console.log('unknown ', unknown);
+console.log('unknown typeof', typeof unknown);
+
+const never: undefined = undefined;
```

Comment:
let never: undefined;

Reply:
I thought I heard you were saying something about that but the zoom was lagging and screenshare wouldn't load for me.